next/components/AppLayout/MobileNavigation/MobileHeader.tsx

```diff
@@ -18,14 +18,15 @@ const TITLE_CLASSES =
   'flex h-[30px] min-w-fit items-center border-r border-border-dark px-[7px] py-[2px]'
 
 const MobilHeader = ({ menus }: HeaderProps) => {
-  const [isOpen, setOpen] = useState(false)
+  const [isMenuOpen, setMenuOpen] = useState(false)
   const router = useRouter()
   const { t } = useTranslation('common')
+  const [isSearchOpen, setSearchOpen] = useState(false)
 
   // close mobile header on route change
   useEffect(() => {
-    router.events.on('routeChangeStart', () => setOpen(false))
-    return () => router.events.off('routeChangeStart', () => setOpen(false))
+    router.events.on('routeChangeStart', () => setMenuOpen(false))
```

Comment:
https://nextjs.org/docs/api-reference/next/router#usage-6

Unsubscribe is missing.